Formula/wasm-micro-runtime.rb

```diff
@@ -0,0 +1,42 @@
+class WasmMicroRuntime < Formula
+  desc "WebAssembly Micro Runtime (WAMR)"
+  homepage "https://github.com/bytecodealliance/wasm-micro-runtime"
+  url "https://github.com/bytecodealliance/wasm-micro-runtime/archive/refs/tags/WAMR-05-18-2022.tar.gz"
+  version "2022.05.18"
+  sha256 "c71f571f99c7028fd7595126ec25d6df4bc9ba62702bb88cbd5ea8a29e477f12"
+  license "Apache-2.0"
+  head "https://github.com/bytecodealliance/wasm-micro-runtime.git", branch: "main"
+
+  depends_on "cmake" => :build
+
+  def install
```

Comment:
Let's also make sure this isn't downloading anything during the build:
```suggestion
  def install
    # Prevent CMake from downloading and building things on its own.
    buildpath.glob("**/build_llvm*").map(&:unlink)
    buildpath.glob("**/libc_uvwasi.cmake").map(&:unlink)
```

Reply:
Thank, that is a good idea.